modules/dxp/apps/scim/scim-rest-impl/src/main/java/com/liferay/scim/rest/internal/configuration/ScimClientOAuth2ApplicationConfigurationFactory.java

```diff
@@ -72,7 +72,8 @@ protected void activate(
 							properties);
 
 				_oAuth2Application = _getOrAddOAuth2Application(
-					companyId, scimClientOAuth2ApplicationConfiguration);
+					companyId, scimClientOAuth2ApplicationConfiguration,
+					(long)properties.get("userId"));
```

Comment:
Please change this to `GetterUtil.getLong(properties.get("userId")));` and adjust SF if needed